### DISCUSSION TITLE
Question related to indicator NSi4.1

### DISCUSSION THREAD

**Discussion by hallinpihlatie** [INSPIRE-MIF/helpdesk#163] [Q&A] [answered]
@jescriu & co

I'm posing my question here as it is both related to the INSPIRE Geoportal and the INSPIRE Validator.

The indicator NSi4.1 is 0% for Finland ([https://inspire-geoportal.ec.europa.eu/mr2022_details.html?country=fi](https://inspire-geoportal.ec.europa.eu/mr2022_details.html?country=fi))

I wonder why it isn't 100% as there is only one discovery service described in our national metadata portal and both the metadata and the CSW-service as such pass the relevant tests of the validator (links and test results below)? 

CSW = [paikkatietohakemisto.fi/geonetwork/srv/en/csw?request=GetCapabilities&service=CSW&version=2.0.0&outputFormat=text/xml](https://www.paikkatietohakemisto.fi/geonetwork/srv/en/csw?request=GetCapabilities&service=CSW&version=2.0.0&outputFormat=text/xml&)

Metadata = https://www.paikkatietohakemisto.fi/geonetwork/srv/api/records/6f77b9f1-aa4f-4688-ab32-2e721bcfefd9/formatters/xml?approved=true

[Test results.zip](https://github.com/INSPIRE-MIF/helpdesk/files/12746442/Test.results.zip)

I hope you can help

**Comment by jescriu** [comment]
Dear @hallinpihlatie,
We will look at it as soon as the team have a chance.

**Comment by fabiovinci** [comment, marked as answer]
Dear @hallinpihlatie,

the indicator NSi4.1 is based on the self-declaration of the conformity of the service with respect to the Commission Regulation (EC) No 976/2009.
In your service metadata, you declared conformity to 1089, this is the reason why you have 0%.

![image](https://github.com/INSPIRE-MIF/helpdesk/assets/22212406/4d22385d-19cc-4714-b7d6-d932b05f545b)

**Comment by hallinpihlatie** [comment]
Of course... Thanks for pointing that out, should be an easy fix.
Replaced it now with "976/2009 etc" and changed the date too, assuming it should be 2009-10-19.
It would be useful if the validator would return an error if you have the referring to the wrong legislation in your metadata, four different options, and prone to typo mistakes. Something for you to consider.

**Comment by jescriu** [comment]
Dear @hallinpihlatie, 
Happy that the problem is solved now.
Please propose this improvement in the Validator helpdesk.

**Comment by fabiovinci** [comment]
Dear @hallinpihlatie, 
you can find [here](https://github.com/INSPIRE-MIF/helpdesk-validator/blob/master/examples/NetworkService_WMS_metadata_2.0_example.xml) a sample service metadata containing the correct encoding for the declaration of conformity to 976.

Regarding the validator, the declaration of conformity for service metadata is only "recommended" ([see related section in MD TG](https://github.com/INSPIRE-MIF/technical-guidelines/blob/main/metadata/metadata-iso19139/metadata-iso19139.adoc#4221-conformity)), so it is not tested.
